test/ad.jl

```diff
@@ -24,29 +28,33 @@ end
     @testset "$at" for at in [
         ADTypes.AutoZygote(),
         ADTypes.AutoForwardDiff(),
-        ADTypes.AutoReverseDiff(false),
-        # ADTypes.AutoEnzyme(), # not working now
+        ADTypes.AutoReverseDiff(; compile = false),
+        ADTypes.AutoMooncake(; config=Mooncake.Config()),
```

Comment:
The test was working with Enzyme (and runtime activity) in https://github.com/TuringLang/NormalizingFlows.jl/pull/38/files.

Reply:
Maybe try with `AutoEnzyme(mode=Enzyme.set_runtime_activity(Enzyme.Reverse))`?